src/All/README.md

```diff
@@ -4,6 +4,20 @@
 All Boolean
 ```
 
+## Constructor
```

Comment:
This should go right after Implements and before [Constructor Methods](https://github.com/evilsoft/crocks/pull/252#discussion_r178992967).
Then this should be GTG I believe.